src/api.rs

```diff
@@ -2307,6 +2351,81 @@ pub struct Project {
     pub team: Option<Team>,
 }
 
+#[derive(Debug, Deserialize)]
```

Comment:
We already have all those types in sentry-sdk crate.

Reply:
Hi @kamilogorek I found [`Event`](https://docs.rs/sentry/0.27.0/sentry/types/protocol/v7/struct.Event.html) but I don't think you're talking about this one because it is not good for this JSON response. 
For `EventUser` you were right because we can use `sentry::protocol::User` instead of `EventUserOption`.
For `EventTags` I suggest to keep the `Vec<EventTag>` due the JSON serialization. If we use the same way used for sentry-sdk we have to refactor `tags` in `Vec<BTreeMap<String, String>>` (or `HashMap`) but we always have a BTreeMap (or HashMap) with a tuple `("key", "...")` and a `("value", "...")` cause the nature of the response.